package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@aserto/console-type-definitions",
-  "version": "1.2.0",
+  "version": "2.0.0",
```

Comment:
this doesn't need a major version bump because it doesn't break backwards-compat